code/modules/mob/living/living.dm

```diff
@@ -511,17 +507,15 @@
 	if ((!(L.stat) && !(L.restrained())))
 		resist_grab(L) //this passes L because the proc requires a typecasted mob/living instead of just 'src'
 
-	// Sliding out of a morgue/crematorium
-	if(loc && (istype(loc, /obj/structure/morgue) || istype(loc, /obj/structure/crematorium)))
-		resist_tray(L)
-
 	//unbuckling yourself
 	if(L.buckled && (L.last_special <= world.time) )
 		resist_buckle(L) //this passes L because the proc requires a typecasted mob/living instead of just 'src'
 
-	//Breaking out of a locker?
-	else if(src.loc && (istype(src.loc, /obj/structure/closet)))
-		resist_closet()
+	//Breaking out of an object?
+	else if(src.loc && istype(src.loc, /obj) && (!isturf(src.loc)))
+		if(stat == CONSCIOUS && !stunned && !weakened && !paralysis)
```

Comment:
I'd do that, but then if you were handcuffed and in a locker, you'd be at the end of your rope